.gitignore

```diff
@@ -24,7 +24,6 @@ bosh-lite/deployments/*
 target
 /app-autoscaler-release/
 test-certs
-src/scheduler/src/test/resources/certs/
```

Comment:
Don't we generate certs in here anymore?

Reply:
Yes, but I have moved it to the more specific .gitignore in ./src/scheduler.